### ISSUE TITLE
homepage - hover style for buttons fails colour contrast

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#127]
i thought button colour should just change to darker colour... rather then text colour changing

![homepage_-_hover_style_for_buttons_fails_colour_contrast](https://user-images.githubusercontent.com/2722522/65626600-9bfb4d80-dfc5-11e9-9ece-5b403ffeab5e.png)

**Comment by enginehouse** [comment]
Hmmm @Dan-KeduTesting  i am not seeing that - could you refresh it and try again?

**Comment by enginehouse** [comment]
<img width="928" alt="Screenshot 2019-09-25 at 19 11 33" src="https://user-images.githubusercontent.com/5637994/65627937-4ffdd800-dfc8-11e9-8635-95b7628fde69.png">
